meseq.py

```diff
@@ -330,14 +338,23 @@ def draw(self):
         y = 0
         for row in self.matrix.rows:
             y += STEP
-            for i in range(len(row)):
```

Comment:
Is this change only code refactoring, or some bug fix, or some improvement ?
If only code refactoring, I do not see clearly the benefit.